src/client/components/ReviewPost/ReviewEditorComponent/ReviewEditorComponent.js

```diff
@@ -0,0 +1,69 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import Button from '../../Button/Button';
+import unknownUser from '../../../assets/images/portraits/unknown-black.png';
+import './ReviewEditorComponent.style.css';
+
+export default function ReviewEditorComponent({
+  avatarUrl,
+  fullName,
+  placeHolderText,
+  buttonStyle,
+}) {
+  const [inputValue, setInputValue] = useState();
+  const onChange = (event) => {
+    setInputValue(event.target.value);
+  };
+
+  return (
+    <div className="container">
+      <div className="top-part">
+        <div className="title-left">
+          <div className="profile-picture">
+            <div className="circle-mask">
+              <img
+                src={avatarUrl}
+                alt="Portrait of user"
+                className="user-photo"
+              />
+            </div>
+          </div>
+          <div className="user-name">{fullName}</div>
+        </div>
+        <div className="title-right-stars" />
+      </div>
+      <div className="input-part">
+        <textarea
+          value={inputValue}
+          placeholder={placeHolderText}
+          onChange={onChange}
+        />
+      </div>
+      <div className="button-part">
+        <Button
+          buttonName="Send review"
+          style={buttonStyle}
+          type="submit"
+          onClick={() =>
+            console.log(
+              'Waiting for the backend to expose endpoint for reviewing',
+            )
+          }
+        />
+      </div>
+    </div>
+  );
+}
+
+ReviewEditorComponent.propTypes = {
+  avatarUrl: PropTypes.string,
+  fullName: PropTypes.string.isRequired,
+  placeHolderText: PropTypes.string,
+  buttonStyle: PropTypes.string,
```

Comment:
@pdsorensen  To resolve the following error, after changed ```buttonStyle``` prototype to object, and the related modification in Button component problem solved, but Eslint show violatio: ```prop type object is forbidden``` any suggestion? (I will not push changes related to Button) :smile_cat: 

![image](https://user-images.githubusercontent.com/19622435/107885480-dfc45f00-6efa-11eb-8381-d5aba954e71a.png)

Reply:
Can't we remove button styles as a prop argument, and define them in the `ReviewEditorComponent`?

Reply:
@pdsorensen Done, but still have warning, I think this related to Button proptypes, does it? I think we can ignore it for now and finalize this, and try to solve the Button prop problem in another story.
![image](https://user-images.githubusercontent.com/19622435/107941764-7db33a80-6f8a-11eb-8470-4400b383a3f9.png)